### DISCUSSION TITLE
Make things more noticeable

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2944] [Ideas] [answered]
The current version homepage icon looked too small, it is blue like the filterlist name make it even more unobvious, the icon should be bigger and the icon should have a different color.

![Screenshot_20231108-211006_Kiwi Browser](https://github.com/community/community/assets/146954135/4bd046a8-8b4b-4251-9900-88dfaca4b676)

Filterlist information have the same problem too, special comments like title, description, last modified should be highlighted, at least it should support those comments used by uBlock's own filters.

![Screenshot_20231108-211138_Kiwi Browser](https://github.com/community/community/assets/146954135/52d82288-271a-4da7-aeb4-863653cea4ef)
Like these :
![Screenshot_20231108-212542_Kiwi Browser](https://github.com/community/community/assets/146954135/0eb2837c-09e4-43e7-a8b2-daf59b640057)

**Comment by ghost** [comment]
So, this is declined too. Why is this a bad idea ?

**Reply by gorhill** [reply, marked as answer]
This is subjective, I do no want to make changes every time someone want their subjective opinions to be applied to uBO. I also happen to have subjective opinions about how things should look, and since I am the one doing the work, I give priority to mines. Unless there are very good convincing arguments, I just decline, and without further comments usually because this is time-consuming.
